src/Compilers/CSharp/Portable/Emitter/EditAndContinue/CSharpDefinitionMap.cs

```diff
@@ -37,23 +37,9 @@ internal sealed partial class CSharpDefinitionMap : DefinitionMap<CSharpSymbolMa
 
         internal override CommonMessageProvider MessageProvider => CSharp.MessageProvider.Instance;
 
-        protected override VariableSlotAllocator CreateSlotAllocator(
-            CSharpSymbolMatcher symbolMap, 
-            Func<SyntaxNode, SyntaxNode> syntaxMap, 
-            IMethodSymbolInternal previousMethod, 
-            DebugId methodId, 
-            ImmutableArray<EncLocalInfo> previousLocals, 
-            IReadOnlyDictionary<int, KeyValuePair<DebugId, int>> lambdaMap,
-            IReadOnlyDictionary<int, DebugId> closureMap, 
-            string stateMachineTypeNameOpt, 
-            int hoistedLocalSlotCount, 
-            IReadOnlyDictionary<EncHoistedLocalInfo, int> hoistedLocalMap,
-            int awaiterSlotCount, 
-            IReadOnlyDictionary<ITypeReference, int> awaiterMap)
+        protected override ILambdaSyntaxHelper GetLambdaSyntaxHelper()
```

Comment:
Perhaps we can use ```=>``` syntax here. BTW, the previous line implements a similar pattern for error messages.